src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletRPCController.cs

```diff
@@ -120,15 +120,20 @@ public bool LockWallet()
 
         [ActionName("sendtoaddress")]
         [ActionDescription("Sends money to an address. Requires wallet to be unlocked using walletpassphrase.")]
-        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest)
+        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest, decimal fee = 0, bool isSegwit = false)
         {
-            WalletAccountReference account = this.GetWalletAccountReference();
+            if (fee <= 0)
+            {
+                fee = this.FullNode.Network.MinTxFee;
+            }
 
             TransactionBuildContext context = new TransactionBuildContext(this.FullNode.Network)
             {
                 AccountReference = this.GetWalletAccountReference(),
                 Recipients = new[] { new Recipient { Amount = Money.Coins(amount), ScriptPubKey = address.ScriptPubKey } }.ToList(),
-                CacheSecret = false
+                CacheSecret = false,
+                TransactionFee = Money.Coins(fee),
```

Comment:
I suppose this is in BTC units?

Reply:
wait TransactionFee  was this null or zero otherwise?

Reply:
> I suppose this is in BTC units?
Yep. I think we should have a better way to do this in the future when we have the fee estimate stuff in blockcore.

>wait TransactionFee was this null or zero otherwise?
Yeah, I don't thing this call was used much until now.